packages/website/package.json

```diff
@@ -22,22 +22,27 @@
     "@tanstack/react-query": "^5.0.0",
     "@tanstack/react-query-devtools": "^5.0.3",
     "@types/mdx": "^2.0.9",
+    "algoliasearch": "v5",
     "autoprefixer": "10.4.15",
+    "axios": "1.7.7",
```

Comment:
Needed?

Reply:
using to create algolia client. Technically not needed but i like the api it provides vs working with fetch.

Reply:
removed